### DISCUSSION TITLE
Connection failed (HTTPSRequest.ino example)

### DISCUSSION THREAD

**Discussion by Larzu** [esp8266/Arduino#8759] [Q&A]
I had been wondering reason behind Connection failed in HTTPSRequest.ino example.  All advices are welcome.

The latest version 3.0.2 is used.

---clip ---
10:06:27.852 -> Connecting to api.github.com
10:06:27.852 -> Using certificate: 
10:06:27.852 -> -----BEGIN CERTIFICATE-----
10:06:27.852 -> MIIDxTCCAq2gAwIBAgIQAqxcJmoLQJuPC3nyrkYldzANB.........................................
-clip - 
10:06:27.989 -> -----END CERTIFICATE-----
10:06:27.989 -> 
10:06:31.637 -> Connection failed
---- clip ----

**Comment by amenoteph** [comment]
Hello 
I have the same pb. Connection failed each time  for http requests and also to connect at SMTP.  This pb occured just on new compils for programs working well before janv 2023. I try also with Github examples with the same bad  results. After many tests with a lot of differents nodeMcu , I reload an old version : 2.7.3 instead of the latest 3.1.1 and it's working well now !!! So there is something wrong on the version 3. (Sorry for my english very approximative).
 Cordialement
